Updates for FO76 and FO76 PTS as of Feb 2 2025 version 0.2.1707.0

eckserah opened the pull request:
Add XRLL which is reference lighting (same structure as XCLL which is cell lighting) and XRL2 which is reference lighting template (same as Cell LTMP which is cell lighting template) to REFR and all REFR records
Add FTAG Form Tags to GLOB, NPC_, PACK and RACE
Add PSCT, PSSC, PFDX, and PFDZ to PROJ
Add PBAP to BPTD
Add LVIT to LVLI
Add ACTV to EMOT (Currently unused)
Add PLYT Player Title record with subrecords of EDID, XALG, ANAM, BNAM, PTPR, PTSU, PTDS, and conditions.
Add Telegraph Data WTDT, WTDA, and WTDS to WEAP
Update header version to 224.

Updates to allow to work with the PTS as of Feb 2 2025. Version 0.2.1707.0
Add Condition functions for GetIsDisguisedWithKeyword and GetIsPlayerGhoul
Add entry points for Instant Reload Clip On Bash, Mod Attack Damage On Striking Appendage, Apply Spell On Actor When Limb Crippled, Mod Rads to Health Mult, Mod Rads to Radshield Mult, and Mod Weak Body Part Damage Mult
Add what seems to be entry check information for FACT. Has FCNR, FCRF, FCOR, FCBM, CITC, CTDA, and FCAF
Add FULL to TXST
Add DESC to EXPL
Add KNAM (Keyword) to OTFT
Add Unknown data at the end of QUST DATA
Add QUST QUCF
Add LOUT LMML
WEAP WSAM is Sneak Attack Multiplier

eckserah committed:
Updates for fallout 76

Add XRLL which is reference lighting (same structure as XCLL which is cell lighting) and XRL2 which is reference lighting template (same as Cell LTMP which is cell lighting template) to REFR and all REFR records
Add FTAG Form Tags to GLOB, NPC_, PACK and RACE
Add PSCT, PSSC, PFDX, and PFDZ to PROJ
Add PBAP to BPTD
Add LVIT to LVLI
Add ACTV to EMOT (Currently unused)
Add PLYT Player Title record with subrecords of EDID, XALG, ANAM, BNAM, PTPR, PTSU, PTDS, and conditions.
Add Telegraph Data WTDT, WTDA, and WTDS to WEAP
Update header version to 224.

eckserah committed:
Updates for FO76

Updates to allow to work with the PTS as of Feb 2 2025. Version 0.2.1707.0
Add Condition functions for GetIsDisguisedWithKeyword and GetIsPlayerGhoul
Add entry points for Instant Reload Clip On Bash, Mod Attack Damage On Striking Appendage, Apply Spell On Actor When Limb Crippled, Mod Rads to Health Mult, Mod Rads to Radshield Mult, and Mod Weak Body Part Damage Mult
Add what seems to be entry check information for FACT. Has FCNR, FCRF, FCOR, FCBM, CITC, CTDA, and FCAF
Add FULL to TXST
Add DESC to EXPL
Add KNAM (Keyword) to OTFT
Add Unknown data at the end of QUST DATA
Add QUST QUCF
Add LOUT LMML
WEAP WSAM is Sneak Attack Multiplier

Kuroitsune commented:
WSAM makes sense now. Figured all four of the new signatures were related to each other but it did lack the WTD prefix. Should've known.

Didn't think about ANAM and BNAM being male and female titles since none of the values were gendered. Makes more sense than Prefix and Suffix.

eckserah commented:
Well it's not part of the telegraph data. That's used for bosses when they are using that weapon in order to telegraph that an attack is coming. WSAM is for the sneak attack multiplier to modify it from being the base 2.0 to something higher for the amount of damage you do while stealthed

eckserah committed:
Updates for FO76

Add additional condition functions and weapon property types
Add record flags 2 and 9 to BOOK ADDN OTFT and PLYT
Add NULL as a valid record type for ALDN display name and RDES
Add AFOH to default objects
